completion/nicematrix.cwl

```diff
@@ -277,8 +286,8 @@ rules/width=##L
 rules/color=#%color
 #endkeyvals
 
-\SubMatrix{delim1}{i-j}{k-l}{delim2}#/NiceTabular,NiceTabular*,NiceTabularX,NiceArray,pNiceArray,bNiceArray,BNiceArray,vNiceArray,VNiceArray,NiceMatrix,pNiceMatrix,bNiceMatrix,BNiceMatrix,vNiceMatrix,VNiceMatrix,NiceArrayWithDelims
-\SubMatrix{delim1}{i-j}{k-l}{delim2}[options%keyvals]#/NiceTabular,NiceTabular*,NiceTabularX,NiceArray,pNiceArray,bNiceArray,BNiceArray,vNiceArray,VNiceArray,NiceMatrix,pNiceMatrix,bNiceMatrix,BNiceMatrix,vNiceMatrix,VNiceMatrix,NiceArrayWithDelims
+\SubMatrix{delim1%delim}{i-j}{k-l}{delim2%delim}#/NiceTabular,NiceTabular*,NiceTabularX,NiceArray,pNiceArray,bNiceArray,BNiceArray,vNiceArray,VNiceArray,NiceMatrix,pNiceMatrix,bNiceMatrix,BNiceMatrix,vNiceMatrix,VNiceMatrix,NiceArrayWithDelims
+\SubMatrix{delim1%delim}{i-j}{k-l}{delim2%delim}[options%keyvals]#/NiceTabular,NiceTabular*,NiceTabularX,NiceArray,pNiceArray,bNiceArray,BNiceArray,vNiceArray,VNiceArray,NiceMatrix,pNiceMatrix,bNiceMatrix,BNiceMatrix,vNiceMatrix,VNiceMatrix,NiceArrayWithDelims
```

Comment:
Currently it seems texstudio doesn't support multiple keyvals fields, either with the same or different keyvals list, in a single command.

Also the format should be
```
\SubMatrix{delim1%keyvals}...

#keyvals:\SubMatrix
...
#endkeyvals
```
This syntax also shows there's no way to specify different keyvals list for different arguments in a single command.